docs/source/configuration/authn-jwt.mdx

```diff
@@ -156,6 +158,8 @@ Below are 2 example [Rhai script](../customizations/rhai/) customizations that d
 
 ### Example: Forwarding claims to subgraphs
 
+> ⚠️ This example forwards JWT claims via headers. When combined with [header propagation](./header-propagation/) this can introduce a security risk. If you're relying on header-based forwarding, **make sure not to propagate matching headers**.
```

Comment:
this requires an example, otherwise readers won't know what's the actual risk

Reply:
Example added—though I didn't go in-depth in mitigation. The idea was to add a quick-fix before #3224 can be handled, so I don't want to add a bunch of potential mitigations in the example (e.g., enforcing claim presence, deleting matching headers)